### PR TITLE
Adding Info from Amelia Gann

### LINKED ISSUE #1064: Amelia Gann bio submission
End user submitted issue from page: [/collect/person/](https://history.newtheatre.org.uk/collect/person/)
---
# 'Submit an almni bio' form submission

Field | Data
----- | ----
Name | Amelia Gann
Grad Year | 2015
Course | English BA

## Bio2 (Post-graduation)

Currently studying at Academy of Live and Recorded Arts, London
MA Acting (postgrad) Sept 2017 - December 2018

Field | Data
------| ----
Checked careers | Actor, 
Other careers | 

## Awards

Ritual Slaughter won Festgoers award at NSDF 

## Contact Preferences

Are we allowed to facilitate contact to this alumnus? **Yes**

## Attempted File Generation

```
---
title: Amelia Gann
course:
  - English BA
graduated: 2015
contact_allowed: true
careers:
  - Actor


links: *fill me out

award: *fill me out
Ritual Slaughter won Festgoers award at NSDF 
---




Currently studying at Academy of Live and Recorded Arts, London
MA Acting (postgrad) Sept 2017 - December 2018

```
**

### PR DESCRIPTION
**Pull request by nhyty**
Closes #1064

### CODE REVIEW

File: _people/amelia_gann.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 title: Amelia Gann
 headshot: trLKLFk
 gender: female
-submitted: false
+submitted: 2018-01-18
+contact_allowed: true
 graduated: 2015
+course:
+ - English
 ---
```
Comment (wjdp) [inline _people/amelia_gann.md]:
```
careers:
  - Actor
```
?

File: _people/amelia_gann.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+Currently studying at Academy of Live and Recorded Arts, London
+MA Acting (postgrad) Sept 2017 - December 2018
```
Comment (wjdp) [inline _people/amelia_gann.md]: Line break before to make new paragraph.

Comment (nhyty) [reply _people/amelia_gann.md]: Ritual comment already there, is down as award or tour on the show file

Comment (wjdp) [reply _people/amelia_gann.md]: Ah, ignore me then 😄

**Pull request merged**